react/src/components/AssetDetail/AssetDetail.tsx

```diff
@@ -0,0 +1,172 @@
+    geoapiUrl + '/assets/' + selectedFeatureAsset?.path;
+
+  const fileType: string | undefined = selectedFeatureAsset?.asset_type;
+
+  const AssetRenderer = React.memo(
+    ({
+      type,
+      source,
+    }: {
+      type: string | undefined;
+      source: string | undefined;
+    }) => {
+      switch (type) {
+        case 'image':
+          return <img src={source} alt="Asset" loading="lazy" width="100%" />;
+        case 'video':
+          return (
+            <video src={source} controls preload="metadata" height="90%" />
+          );
+        case 'point_cloud':
+          /*TODO Add pointcloud */
+          return <div> source={source}</div>;
+        case 'questionnaire':
+          /*TODO Add questionnaire */
+          return <div> source={source}</div>;
+        default:
+          return null;
+      }
+    }
+  );
+
+  return (
+    <div className={styles.root}>
+      <div className={styles.topSection}>
+        <FeatureIcon featureType={fileType as FeatureTypeNullable} />
+        {selectedFeature?.id}
+        <Button type="link" iconNameAfter="close" onClick={closePanel}></Button>
+      </div>
+      <div className={styles.middleSection}>
+        {fileType ? (
+          <>
+            <Suspense fallback={<LoadingSpinner />}>
+              <AssetRenderer type={fileType} source={featureSource} />
+            </Suspense>
+            <Button /*TODO Download Action */>Download</Button>
+          </>
+        ) : (
+          <>
+            <SectionMessage type="info">Feature has no asset.</SectionMessage>
+
+            <Button type="primary" /* TODO Add asset to a feature */>
+              Add asset from DesignSafe
+            </Button>
+          </>
+        )}
+      </div>
+      <div className={styles.bottomSection}>
+        <div className={styles.metadataTable}>
+          <table>
+            <thead>
+              <tr>
+                <th colSpan={2}>Metadata</th>
+              </tr>
+            </thead>
+            <tbody>
+              {selectedFeature?.properties &&
+              Object.keys(selectedFeature.properties).length > 0 ? (
+                Object.entries(selectedFeature.properties)
+                  .filter(([key]) => !key.startsWith('_hazmapper'))
+                  .sort(([keyA], [keyB]) => keyA.localeCompare(keyB)) // Alphabetizes metadata
+                  .map(([propKey, propValue]) => (
+                    <tr key={propKey}>
+                      <td>{_.startCase(propKey)}</td>
+                      <td>
+                        {propKey.startsWith('description') ? (
+                          <code>{propValue}</code>
+                        ) : (
+                          _.trim(JSON.stringify(propValue), '"')
+                        )}
+                      </td>
+                    </tr>
+                  ))
+              ) : (
+                <tr>
+                  <td colSpan={2}>
+                    There are no metadata properties for this feature.
```

Comment:
```suggestion
                    There are no metadata properties.
```
I like the message; here is a suggestion so we can avoid saying "feature"